README.md

```diff
@@ -95,6 +95,9 @@ Please note that a public api key (`choisirgeoportail`) is used by default and c
 
 4 aliases are also provided for common Geoportail resources : `GeoportailFrance`, `GeoportailFrance.orthos`, `GeoportailFrance.ignMaps` and `GeoportailFrance.parcels` (See index.html demo).
 
+### Stadia Maps
```

Comment:
@jieter i think this part of the PR was useful in order to warn people that stadia api require an account linked to a whitelist of domains, any thoughts about this ?